pkg/vm/engine/disttae/logtailreplay/partition_state.go

```diff
@@ -474,6 +473,19 @@ func (p *PartitionState) HandleMetadataDelete(ctx context.Context, input *api.Ba
 	})
 }
 
+func (p *PartitionState) GetBlock(blockID types.Blockid) *BlockEntry {
```

Comment:
Is this method being used? If not, remove it.

Reply:
Done